### ISSUE TITLE
[JIT] dropout fails on legacy mode

### ISSUE THREAD

**Issue by xwang233** [pytorch/pytorch#35446]
## To Reproduce

On pytorch master build (1.5.0a0+efbd6b8), run 

```python 
python test/test_jit_legacy.py -k test_dropout_cuda 
``` 

The test fails. 

## Expected behavior

However, 

```python
python test/test_jit.py -k test_dropout_cuda
```

passes. 

## Environment

 - PyTorch Version (e.g., 1.0): 1.5.0a0+efbd6b8
 - OS (e.g., Linux): Fedora 29
 - Python version: 3.7.6
 - CUDA/cuDNN version: 10.1

## Additional context

This may be related to #34361. The test failure may also be related to legacy profiling executer. By commenting out the following line, the legacy test will also pass. 

https://github.com/pytorch/pytorch/blob/efbd6b8533bfa42f9e57d11b453b8a8bf4cf18ea/test/test_jit.py#L168

I'm also not sure if `test/test_jit_legacy.py` runs on pytorch CI? 

cc @ptrblck @jjsjann123

**Comment by suo** [comment]
cc @Krovatkin, @ZolotukhinM, mind taking a look? Looks related to the tests you were working on last week

**Comment by Krovatkin** [comment]
@suo We didn't have a CUDA job running the legacy executor so a few tests regressed. I'm slowly looking into fixing some of them.

**Comment by seemethere** [comment]
We're also experiencing this on the `v1.5.0-rc3`

@Krovatkin, @ZolotukhinM is there a resolution to this?

**Comment by Krovatkin** [comment]
@seemethere didn't get to it. fixed `test_clamp` last week.

**Comment by ZolotukhinM** [comment]
Should we disable the test until it's fixed?

**Comment by gchanan** [comment]
@ZolotukhinM @Krovatkin as @seemethere mentioned, this is failing the release build.  If this is a serious issue, we need to fix it for the release.  If it's not a serious issue, it's okay to disable the tests.  Please advise as to the severity.

**Comment by Krovatkin** [comment]
@gchanan we decided to enable the profiling executor for 1.5, so the legacy executor won't be the default jit mode anymore. We should probably be okay to disable the test. I'll take a look at what exactly the issue is with this test either today or tomorrow.

**Comment by gchanan** [comment]
updating milestone to 1.5.1.

@Krovatkin any update?  Did we end up disabling the test?

**Comment by gchanan** [comment]
going to tentatively move to 1.6.0 milestone because I haven't seen any movement on this.

**Comment by seemethere** [comment]
Going to be removing this from the `1.6.0` milestone since there hasn't been any update.

**Comment by janeyx99** [comment]
TorchScript has been deprecated in favor of torch.export as of 2.11. https://pytorch.org/blog/pytorch-2-11-release-blog/ Closing this issue as we are not further developing on it--please ping if you believe this issue was erroneously closed.
